app/connector/kafka/src/main/resources/META-INF/syndesis/connector/kafka.json

```diff
@@ -140,11 +111,11 @@
       "secret": false,
       "type": "typeahead"
     },
-    "transportProtocol": {
```

Comment:
We need a migration script for this.

Reply:
Thank you, Zoran. I created FUSEDOC-4596 to address related documentation updates...

Reply:
I will revert this change, it was mostly to reflect the parameter name in camel-kafka component, but not worth to spend too much on this renaming.